docs/use-mainnet/linea-xp.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Linea Voyage XP 
+sidebar_position: 5
+---
+
+## What is Linea Voyage XP?
+
+Similar to video games, where your character can gain experience points (XP) as you go through the game, Voyage XP represents your journey and experience in the Linea ecosystem. The Voyage XP tokens will be a custom version of ERC-20 and will be visible by default in your MetaMask wallet **Voyage XP are non-transferrable soulbound tokens that are distributed to recognize the community’s extraordinary contribution toward the growth of the Linea ecosystem and do not have any monetary value!**
```

Comment:
How are they visible by default? Afaik token detection is not supported on Linea. Unless it's being hard-coded in somehow.

Reply:
not sure, this is just what I grabbed from the blog post. i can add more info if we got more info